pytaskflow/dashboard/controllers/core.py

```
@@ -0,0 +1,65 @@
+from litestar.datastructures import State
+
+from pytaskflow.client import Client
+
+
+def get_client(state: State) -> Client:
+    return state.client
+
+
+class CoreController(Controller):
+    path = "/"
+    dependencies = {"client": Provide(get_client, sync_to_thread=False)}
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)